test/test_cloudwatch_plugin.py

```diff
@@ -0,0 +1,59 @@
+import gevent
+from locust import events
+from locust import HttpUser, task, between
+from locust.env import Environment
+from locust.stats import stats_printer, stats_history
+from locust.log import setup_logging
+from locust_plugins.listeners.cloudwatch import CloudwatchAdapter
+from unittest import TestCase
+
+
+setup_logging("INFO", None)
+
+class CloudwatchMock:
+    def __init__(self):
+        self.call_count = 0
+        self.metrics_dict = {}
+
+
+    def put_metric_data(self, Namespace, MetricData):
+        self.call_count += 1
+        self.metrics_dict[Namespace] = MetricData
+        
+
+
+cw = CloudwatchMock()
+
+def on_locust_init(environment, **_kwargs):
+    CloudwatchAdapter(environment, "MyExampleService", "perf", cw)
+
+
+events.init.add_listener(on_locust_init)
```

Comment:
this modifies the "global" events object.

Instead, you can do `env.events.init.add_listener(on_locust_init)` after line 44.

Reply:
Done the changes